### ISSUE TITLE
Magic UV won't do anything

### ISSUE THREAD

**Issue by GrindingAgony** [nutti/Magic-UV#153]
System Information - Windows 10, Blender 3.0.1

Expected to perform much like the Magic UV in Blender 2.79, which I've used for years.

I can't get anything to work, even on a simple UV sphere, all marked out with seams & unwrapped into 4 sections.
Any Magic UV command is continually blocked by the message "All UV's are not separated"
Well, they are separated, no overlaps. Everything's fine.

I've now tried this on two completely separate machines, both Win 10, Blender 3.0.1 & Magic UV isn't working.

New file, one UV sphere, mark with seams, unwrap, & Magic UV isn't working.

**Comment by nutti** [comment]
@GrindingAgony 

Currently below two features does not work correctly which is already mentioned #79 .

* Smooth UV
* Align UV

I need a time to fix this issue because only specific case is supported now.

**Comment by nutti** [comment]
This issue is duplicate of #79.
Close it for now.
